src/components/BreedsList.vue

```diff
@@ -1,31 +1,22 @@
 <template>
   <section>
+    <!-- :noMatches="errorState.class" -->
     <search-form
       :isDisabled="isInputDisabled"
       @search="getSearchResults"
     ></search-form>
 
     <div class="result__container">
-      <p v-if="isLoading">The Data is Loading ...</p>
-      <p v-else-if="!isLoading && error" class="result__error">{{ error }}</p>
-      <p v-else-if="isTimeOut && !displayedBreeds.length">
-        The request takes longer than expected...
+      <p class="result__message" v-if="loadingState">{{ loadingState }}</p>
+      <p class="result__message" v-else-if="receivedDataState">
+        {{ receivedDataState }}
```

Comment:
Zdej sem popravil kodo, tako da je malo bolj razumljiva. Vse message sem nastavil že pri fetchanju in potem ločil  `loadingState `in `receivedDataState `computed prop, kjer pri loading prikaže samo message, ki se zgodi ko se loudajo podatki+error, pri receivedDataState pa ko podatke že ima, torej, če dobi prazno dato, oz ko uporabnik išče in ne najde zadetkov+error
kako se ti zdi?

Reply:
mal mi je čudn, ker se mi ko hočem posearchat in med tem ko fetcha, pokaže `no data found` error msg.. namest loadinga
<img width="823" alt="Screenshot 2023-04-20 at 08 10 23" src="https://user-images.githubusercontent.com/3824010/233274515-0bb05490-b3b7-4ba5-8fb3-dbfe95db74c3.png">

je pa boljš kot je blo prej, samo še par detailov je treba spimpat ;)